mmv1/products/cloudrun/Service.yaml

```diff
@@ -850,6 +850,29 @@ properties:
                           name: 'sizeLimit'
                           description: |-
                             Limit on the storage usable by this EmptyDir volume. The size limit is also applicable for memory medium. The maximum usage on memory medium EmptyDir would be the minimum value between the SizeLimit specified here and the sum of memory limits of all containers in a pod. This field's values are of the 'Quantity' k8s type: https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/. The default is nil which means that the limit is undefined. More info: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir.
+                    - !ruby/object:Api::Type::NestedObject
+                      name: csi
+                      description: |-
+                        A filesystem specified by the Container Storage Interface (CSI).
+                      min_version: beta
+                      properties:
+                        - !ruby/object:Api::Type::String
+                          name: 'driver'
+                          required: true
+                          description: |-
+                            Unique name representing the type of file system to be created. Cloud Run supports the following values:
+                              * gcsfuse.run.googleapis.com: Mount a Google Cloud Storage bucket using GCSFuse
+                        - !ruby/object:Api::Type::String
+                          name: 'readOnly'
+                          default_from_api: true
+                          description: |-
+                            If true, all mounts created from this volume will be read-only.
+                        - !ruby/object:Api::Type::KeyValuePairs
+                          name: 'volumeAttributes'
+                          description: |-
+                            Driver-specific attributes. The following options are supported for available drivers:
+                              * gcsfuse.run.googleapis.com
+                                * bucketName: the name of the Cloud Storage Bucket that backs this volume. The Cloud Run Service identity must have access to this bucket.
```

Comment:
it might be good to specify the allowed format(s) of the bucketName - is it a full id or just the unique short name?

Reply:
It's just the unique name. I'm not really sure how to specify a format for that.

Reply:
I think this is probably fine in that case.